specs/player_spec.rb

```diff
@@ -0,0 +1,155 @@
+require_relative 'spec_helper'
+
+describe "Player class" do
+
+  before do
+    @person = Scrabble::Player.new("Ursula")
+    @tilebag = Scrabble::TileBag.new
+  end
+
+  describe "@name instance variable" do
+    it "Returns the value of @name" do
+      @person.name.must_equal "Ursula"
+    end
+  end
+
+  describe "@plays Array" do
+    it "Returns an Array of words played" do
+      @person.plays.must_be_kind_of Array
+    end
+  end
+
+  describe "play(word) method" do
+    it "Does not add empty words to plays Array" do
+      @person.play("")
+      @person.plays.must_be_empty
+    end
+
+    it "Adds words played to plays Array" do
```

Comment:
I really like this test! It's generic enough that if we change unrelated things, like the number of tiles that are drawn by `Player#draw_tiles`, this test will still work and properly test the specific behavior we care about here (updating the `@plays` array).